pandas/core/arrays/period.py

```diff
@@ -489,15 +493,19 @@ def _time_shift(self, periods, freq=None):
     def _box_func(self, x) -> Union[Period, NaTType]:
         return Period._from_ordinal(ordinal=x, freq=self.freq)
 
+    @doc(**_shared_doc_kwargs, other="PeriodIndex", other_name="PeriodIndex")
     def asfreq(self, freq=None, how: str = "E") -> "PeriodArray":
         """
-        Convert the Period Array/Index to the specified frequency `freq`.
+        Convert the {klass} to the specified frequency `freq`.
+
+        Equivalent to applying :meth:`pandas.Period.asfreq` with the given arguments
+        to each :class:`~pandas.Period` in this {klass}.
 
         Parameters
         ----------
         freq : str
             A frequency.
-        how : str {'E', 'S'}
+        how : str {{'E', 'S'}}, default 'E'
```

Comment:
Thanks @plammens - I haven't tried running this yet, but why are you doubling the brackets here? Single brackets seems consistent with the docstring guide https://pandas.pydata.org/docs/development/contributing_docstring.html